t/parse/grammar_anythingany.tex

```diff
@@ -0,0 +1,43 @@
+\documentclass{article}
+\usepackage{latexml}
+\usepackage{amsmath}
+\begin{document}
+
+% Parse rule tree: 
+% 1 AnythingAny 
+% 2  MODIFIEROP Expression  
+% 3  MODIFIEROP ( SignedTerm moreTerms addExpressionModifier )  
+% 4  MODIFIEROP (( AddOp Term ) moreTerms addExpressionModifier )  
+% 5  MODIFIEROP ((( ADDOP addOpDecoration ) ( Factor moreFactors )) moreTerms addExpressionModifier )  
+% 6  MODIFIEROP ((( ADDOP addOpDecoration ) (( NUMBER addScripts ) ( Factor moreFactors ))) moreTerms addExpressionModifier )  
+% 7  MODIFIEROP ((( ADDOP addOpDecoration ) (( NUMBER addScripts ) (( preScripted doubtArgs ) moreFactors ))) moreTerms addExpressionModifier )  
+% 8  MODIFIEROP ((( ADDOP addOpDecoration ) (( NUMBER addScripts ) ((( <matchrule:$arg[0]> addScripts ) doubtArgs ) moreFactors ))) moreTerms addExpressionModifier )  
+% lex  MODIFIEROP ((( ADDOP ) (( NUMBER ) ((( <matchrule:$arg[0]> )))))) 
+
+$\mod -2y$
```

Comment:
My idea of going forward would be to keep all of these formulas, drop the parse steps, and maybe keep the last "lex" level, explaining the *intended* grammatical roles of each lexeme in the expression. So rework the first example to:

```tex
% MODIFIEROP (ADDOP NUMBER UNKNOWN)
% Intent: modifier operators can be formula roots
$\mod -2y $
```